<?xml version="1.0"?>
<launch>
    <node pkg="tf" type="static_transform_publisher" name="cameara_fixed" args="-0.5 0.4 0.6 0.0 3.14159 0.0 world camera_frame 20" />

    <node name="fake_ar_publisher" pkg="fake_ar_publisher" type="fake_ar_publisher_node" output="screen"/>
    <node name="vision_node" pkg="titan_ur5e" type="vision_node" output="screen"/>
    <!-- <node name="descartes_node" pkg="app_core" type="descartes_node" output="screen"/> -->

    <!-- <node name="app_node" pkg="app_core" type="app_node" output="screen">
        <param name="base_frame" value="world"/>
    </node> -->

</launch>

